UPGRADING.md

```diff
@@ -10,6 +10,18 @@ gem 'grape-swagger'
 gem 'grape-swagger-entity'
 ```
 
+`add_swagger_documentation` has changed from
+``` ruby
+  add_swagger_documentation \
+    api_version: 'v1'
+```
+to
+
+``` ruby
+  add_swagger_documentation \
+    version: 'v1'
```

Comment:
did you tried it? think this isn't possible anymore here, cause the API version should be specified in grape,
here one could only add `doc_version`